README.md

```diff
@@ -3,40 +3,58 @@
 
 TripleA is a free game engine that runs on open source and is community supported.
 
-Installing TripleA and Playing
-==============================
-- Download and install TripleA: http://triplea-game.github.io/download/
+# Installing TripleA and Playing
+
+- Download and install TripleA: http://www.triplea-game.org/download/
```

Comment:
`www.triplea-game.org` should be just `triplea-game.org`

Reply:
442d8fa9f